src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -113,6 +113,23 @@ public static void selectMacroActionOfOutputSchemaProperty() {
     WaitHelper.waitForElementToBeHidden(CdfSchemaLocators.schemaActionType("macro"));
   }
 
+  /**
+   * Change data type of fields in output schema
+   *
+   * @param jsonFieldList Property key in the Plugin Properties file whose value
+   *                              is the json array with field name and datatype
+   *                              ({@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE})
+   */
+  public static void changeDataTypesInOutputSchema(String jsonFieldList) {
+    Map<String, String> outputSchemaFieldList =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(jsonFieldList));
+    for (Map.Entry<String, String> entry : outputSchemaFieldList.entrySet()) {
+      ElementHelper.clickOnElement(CdfSchemaLocators.outputSchemaDataTypeDropdown(entry.getKey()));
+      SeleniumDriver.getDriver()
+        .findElement(CdfSchemaLocators.outputSchemaDataTypeOption(entry.getKey(), entry.getValue())).click();
```

Comment:
Can we use `ElementHelper` here?

Reply:
No. Element helper click method first waits for element to be clickable, then scroll to that element and then click.
Output schema datatype options list element is not scrollable. We get exception while scrolling -
`element not interactable: [object HTMLOptionElement] has no size and location`
So direct click action is used.